modules/README.markdown

```diff
@@ -134,4 +134,17 @@ closure arguments.
 	or multiple `excludes` declarations), write them on separate lines.
 	* Order assignments alphabetically, leaving an empty line after multiple
 	calls to the same method (e.g., after multiple `dependsOn` declarations) or if
-	the assignment has a closure.
\ No newline at end of file
+	the assignment has a closure.
+
+## Subrepositories
+
+### `gradle.properties`
+
```

Comment:
@Ithildir I think it would be helpful to add a brief description of what we mean `gradle.properties` of a subrepository. Do you think the following would be OK?

> The following settings are available to you in the `gradle.properties` file of a Liferay subrepository (e.g., [com-liferay-journal](https://github.com/liferay/com-liferay-journal)).